lib/internal/util.js

```diff
@@ -9,13 +12,21 @@ exports.setHiddenValue = binding.setHiddenValue;
 // All the internal deprecations have to use this function only, as this will
 // prepend the prefix to the actual message.
 exports.deprecate = function(fn, msg) {
-  return exports._deprecate(fn, `${prefix}${msg}`);
+  return exports._deprecate(fn, msg);
 };
 
 // All the internal deprecations have to use this function only, as this will
 // prepend the prefix to the actual message.
-exports.printDeprecationMessage = function(msg, warned) {
-  return exports._printDeprecationMessage(`${prefix}${msg}`, warned);
+exports.printDeprecationMessage = function(msg, warned, ctor) {
+  if (warned || noDeprecation)
+    return true;
+  const dw = Warning('DeprecationWarning', msg,
```

Comment:
please rename to something more descriptive, doesn't need to be long, but more than `dw`